docs/newsletter.rst

```diff
@@ -4,6 +4,20 @@ Newsletter & Mailing lists
 Write the Docs publishes a monthly newsletter, sharing interesting information
 and ideas that have been shared in our Slack network.
 
+You can see our newsletter archives to get a better idea of the content that we produce:
+
+.. raw:: html
```

Comment:
If someone is feeling enthusiastic they could throw together a custom directive for this. :-D

Reply:
Yea, I thought about a quick include -- definitely should happen.